src/github-cli/install.sh

```diff
@@ -17,11 +18,8 @@ keyserver hkp://keyserver.pgp.com"
 
 set -e
 
-# Clean up
```

Comment:
This PR https://github.com/devcontainers/features/pull/210 intentionally added the cleaning up of old packages at the start and end of the script (for all Features) which makes sense to me. Hence, each Feature cleans up the cache and starts afresh. Do you have any other thoughts where this would not be helpful ?

Reply:
Thanks for linking to the more general PR. I did not realize this pattern spans multiple different features.

I found it odd that simply activating the `github-cli` feature would have side-effects on the apt-get mechanism, even if `apt-get update` wasn't invoked at all. Also, `/var/lib/apt/lists` is an implementation detail of apt-get and I don't think it's healthy for scripts like ours to rely on it in the long run. Finally, I find that cleaning the `/var/lib/apt/lists` cache after the script is done is unnecessary as caches exist for a reason. So my suggestion would be to just do `apt-get update` once, then `apt-get install` a few times to get what we need, and to leave everything else to the system.